src/features/swaps/SwappingScreen.tsx

```diff
@@ -37,7 +36,7 @@ const SwappingScreen = () => {
     // Reset Swap stack to first screen
     navigation.reset({
       index: 0,
-      routes: [{ name: 'Home' }],
+      routes: [{ name: 'SwapScreen' }],
```

Comment:
This should be `Home` since `SwapScreen` is only a modal now

Reply:
<img width="1097" alt="Screenshot 2023-03-17 at 15 12 52" src="https://user-images.githubusercontent.com/37663993/225945208-d2a9ce52-94ca-4ca5-b3a8-af12db44e89b.png">

Reply:
Done!